types/stampit/v3/index.d.ts

```diff
@@ -0,0 +1,525 @@
+     * privileged methods
+     * @returns A new stamp
+     */
+    export function initializers(functions: Init[]): Stamp;
+
+    /**
+     * A shortcut method for stampit().statics()
+     * @param statics Object(s) containing map of property names and values to mixin into each new stamp.
+     * @return A new Stamp.
+     */
+    export function statics(...statics: {}[]): Stamp;
+
+    /**
+     * A shortcut method for stampit().staticProperties()
+     * @param statics Object(s) containing map of property names and values to mixin into each new stamp.
+     * @return A new Stamp.
+     */
+    export function staticProperties(...statics: {}[]): Stamp;
+
+    /**
+     * A shortcut method for stampit().deepStatics()
+     * @param deepStatics The object(s) containing static properties to be
+     * merged
+     * @returns A new stamp
+     */
+    export function deepStatics(...deepStatics: {}[]): Stamp;
+
+    /**
+     * A shortcut method for stampit().staticDeepProperties()
+     * @param deepStatics The object(s) containing static properties to be
+     * merged
+     * @returns A new stamp
+     */
+    export function staticDeepProperties(...deepStatics: {}[]): Stamp;
+
+    /**
+     * A shortcut method for stampit().conf()
+     * @param confs The object(s) containing metadata properties
+     * @returns A new Stamp
+     */
+    export function conf(...confs: {}[]): Stamp;
+
+    /**
+     * A shortcut method for stampit().configuration()
+     * @param confs The object(s) containing metadata properties
+     * @returns A new Stamp
+     */
+    export function configuration(...confs: {}[]): Stamp;
+
+    /**
+     * A shortcut method for stampit().deepConf()
+     * @param deepConfs The object(s) containing metadata properties
+     * @returns A new Stamp
+     */
+    export function deepConf(...deepConfs: {}[]): Stamp;
+
+    /**
+     * A shortcut method for stampit().deepConfiguration()
+     * @param deepConfs The object(s) containing metadata properties
+     * @returns A new Stamp
+     */
+    export function deepConfiguration(...deepConfs: {}[]): Stamp;
+
+    /**
+     * A shortcut method for stampit().propertyDescriptors()
+     * @param descriptors 
+     * @returns A new Stamp
+     */
+    export function propertyDescriptors(...descriptors: {}[]): Stamp;
+
+    /**
+     * A shortcut method for stampit().staticPropertyDescriptors()
+     * @param descriptors 
+     * @returns A new Stamp
+     */
+    export function staticPropertyDescriptors(...descriptors: {}[]): Stamp;
+
+    /**
+     * Take two or more Composables and combine them to produce a new Stamp.
+     * Combining overrides properties with last-in priority.
+     * @param composables Composable objects used to create the stamp.
+     * @return A new Stamp made of all the given composables.
+     */
+    export function compose(...composables: Composable[]): Stamp;
+}
+
+declare module "stampit" {
+    export = stampit;
```

Comment:
This `export = stampit` should really occur at the top level. Remove the `declare module "stampit"` entirely.